packages/p2p-v2/src/routes/AppContent/index.tsx

```diff
@@ -4,15 +4,23 @@ import { useEventListener } from 'usehooks-ts';
 import { useActiveAccount } from '@deriv/api';
 import { Loader, Tab, Tabs } from '@deriv-com/ui';
 import { CloseHeader } from '../../components';
-import { MyProfile } from '../../pages';
+import { MyAds, MyProfile } from '../../pages';
 import './index.scss';
 
 const DEFAULT_TAB = 'buy-sell';
 
 export const routesConfiguration = [
-    { Component: <div> Buy Sell Page </div>, path: 'buy-sell', title: 'Buy / Sell' },
+    { Component: <div> Buy sell page </div>, path: 'buy-sell', title: 'Buy / Sell' },
```

Comment:
nit: did we need to change this 👀

Reply:
oops.. i had put a component there for testing something.. wrote it this way when i change it back.. will correct it.